package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "quickchart",
+    "label": "Quickchart Package",
+    "dependencies": [
+        {
+            "type": "service",
+            "name": "http",
+            "version": "v1.3.3",
+            "instanceId": "shared"
+        }
+    ],
+    "configuration": [
+        {
+            "name": "key",
+            "label": "Key",
+            "type": "text",
+            "description": "Required for QuickChart Pro.",
+            "required": false
+        },
+        {
+            "name": "QUICKCHART_API_BASE_URL",
+            "label": "Quickchart Api Url",
+            "type": "label",
+            "value": "https://quickchart.io"
+        }
+    ],
+    "events": [
+        {
+            "label": "Chart Response",
+            "name": "chartResponse"
+        },
+        {
+            "label": "QR Response",
+            "name": "qrResponse"
+        }
+    ],
+    "metadata": [
+        {
+            "type": "script",
+            "namespace": "api",
+            "path": "/scripts/functions.js"
```

Comment:
```suggestion
            "path": "/scripts/api.js"
```

Reply:
Rename the functions.js file

Reply:
Or if you prefer we can use functions.js for special helpers and api only for the default http methods.